testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractAndroidSantaTrackerSmokeTest.groovy

```diff
@@ -106,6 +104,15 @@ class AbstractAndroidSantaTrackerSmokeTest extends AbstractSmokeTest implements
                 "--add-opens", "jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED"
             )
         }
+        runner.maybeExpectLegacyDeprecationWarning(
```

Comment:
Like the other comment, would be nice if we could reduce the duplication in these expectations